day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,23 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    puts "Add #{toppings} on top of my burrito"
+  end
+
+  def remove_topping (toppings)
+    puts "Remove the #{toppings} from my burrito."
+  end
+
+  def change_protein(protein)
+    @protein = protein
+    puts "Can I switch my protien to #{protein}?"
+  end
```

Comment:
Question to consider: After running methods `add_topping` and/or `remove_topping`, has the `@toppings` array changed?